### PR TITLE
feat(cloudinary): add `secure_url` property

### PR DESCRIPTION
**Pull request by eranshmil**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [x] The changes have been tested successfully.
- [x] A changeset has been created (`npm run changeset`).

### CODE REVIEW

**Review by robingenz — CHANGES_REQUESTED** [review]
Hi @eranshmil,
thank you for your contribution.

According to the Cloudinary docs, there seems to be both: `url` and `secure_url` (see [here](https://cloudinary.com/documentation/upload_images#upload_response))

Feel free to add `secure_url`, but `url` should stay. Please also update the `definitions.ts` file and run `npm run docgen`.

**Comment by EranClientside** [comment]
> Hi @eranshmil, thank you for your contribution.
> 
> According to the Cloudinary docs, there seems to be both: `url` and `secure_url` (see [here](https://cloudinary.com/documentation/upload_images#upload_response))
> 
> Feel free to add `secure_url`, but `url` should stay. Please also update the `definitions.ts` file and run `npm run docgen`.

Yeah, I saw that in the docs, but did it that way just like it's implemented in the [web implementation](https://github.com/capawesome-team/capacitor-plugins/blob/main/packages/cloudinary/src/utils.ts#L49).
Should I align all the properties in the web implementation?

**Comment by eranshmil** [comment]
> Hi @eranshmil, thank you for your contribution.
> 
> According to the Cloudinary docs, there seems to be both: `url` and `secure_url` (see [here](https://cloudinary.com/documentation/upload_images#upload_response))
> 
> Feel free to add `secure_url`, but `url` should stay. Please also update the `definitions.ts` file and run `npm run docgen`.

Yeah, I saw that in the docs, but did it that way just like it's implemented in the [web implementation](https://github.com/capawesome-team/capacitor-plugins/blob/main/packages/cloudinary/src/utils.ts#L49).
Should I align all the properties in the web implementation?

**Comment by robingenz** [comment]
Well spotted! Please add the new property `secure_url` on all platforms. I will replace `url: response.secure_url` with `url: response.url` on the web in a seperate PR.

File: .changeset/mean-maps-shop.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@capawesome/capacitor-cloudinary': patch
```
Comment (robingenz) [inline .changeset/mean-maps-shop.md]:
```suggestion
'@capawesome/capacitor-cloudinary': minor
```

File: .changeset/mean-maps-shop.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+---
+
+feat(cloudinary): add secure url property
```
Comment (robingenz) [inline .changeset/mean-maps-shop.md]:
```suggestion
feat(cloudinary): add `secure_url` property
```

File: packages/cloudinary/README.md [hunk 3]
```diff
@@ -180,6 +180,7 @@ further processing after downloading.
 | **`resourceType`**     | <code><a href="#resourcetype">ResourceType</a></code> | The resource type of the uploaded resource.                                              | 0.0.1 |
 | **`publicId`**         | <code>string</code>                                   | The unique public identifier of the uploaded resource.                                   | 0.0.1 |
 | **`url`**              | <code>string</code>                                   | The url of the uploaded resource.                                                        | 0.0.1 |
+| **`secureUrl`**        | <code>string</code>                                   | The secure url of the uploaded resource.                                                 | 5.0.1 |
```
Comment (robingenz) [inline packages/cloudinary/README.md]:
```suggestion
| **`secureUrl`**        | <code>string</code>                                   | The secure url of the uploaded resource.                                                 | 5.1.0 |
```

File: packages/cloudinary/docs/utils/README.md [hunk 4]
```diff
@@ -70,6 +70,7 @@ Download a file from Cloudinary as a blob.
 | **`resourceType`**     | <code><a href="#resourcetype">ResourceType</a></code> | The resource type of the uploaded resource.                                              | 0.0.1 |
 | **`publicId`**         | <code>string</code>                                   | The unique public identifier of the uploaded resource.                                   | 0.0.1 |
 | **`url`**              | <code>string</code>                                   | The url of the uploaded resource.                                                        | 0.0.1 |
+| **`secureUrl`**        | <code>string</code>                                   | The secure url of the uploaded resource.                                                 | 5.0.1 |
```
Comment (robingenz) [inline packages/cloudinary/docs/utils/README.md]:
```suggestion
| **`secureUrl`**        | <code>string</code>                                   | The secure url of the uploaded resource.                                                 | 5.1.0 |
```

File: packages/cloudinary/src/definitions.ts [hunk 5]
```diff
@@ -160,6 +160,12 @@ export interface UploadResourceResult {
    * @since 0.0.1
    */
   url: string;
+  /**
+   * The secure url of the uploaded resource.
+   *
+   * @since 5.0.1
```
Comment (robingenz) [inline packages/cloudinary/src/definitions.ts]:
```suggestion
   * @since 5.1.0
```

**Review by robingenz — CHANGES_REQUESTED** [review]
Please make these changes and we can merge it. 👍

**Comment by robingenz** [comment]
@eranshmil Thank you for your contribution!

**Pull request merged**